adstb/app.py

```diff
@@ -341,13 +334,26 @@ def _retrieve_abstract_template(self, url):
                 break
             
         if x > end:
-            raise Exception("Cannot find abstract section")
-        
+            raise Exception('Cannot process fetched page, cannot find abstract section for {}'.format(url))
+
         html = html[0:x] + '{{abstract}}' + html[end:]
-        
-        if 'bibcode' in parts and parts['bibcode']:
-            html = html.replace(parts['bibcode'], u'{{bibcode}}')
-        
```

Comment:
this should be kept (and used if it's not already); otherwise links will be wrong

Reply:
updated pr